### ISSUE TITLE
Eckert2016 might have baryon instead of gas masses

### ISSUE THREAD

**Issue by edoaltamura** [SWIFTSIM/velociraptor-comparison-data#66]
I found that the Eckert2016 data might report the *baryon* fraction, as opposed to the gas fractions. When plotting the data from the `.dat` file used here and compare with the original plot (figure 4), even with the same cosmology the data here are higher than the gas fractions in the paper. One explanation could be that those might actually be the baryon fractions (always >= gas fractions) instead of gas.
Another reason why I suspect that is because of the variable name `fb_500` in the conversion script https://github.com/SWIFTSIM/velociraptor-comparison-data/blob/master/data/HaloMassGasFractions/conversion/convertEckert2016.py, which I would normally use for baryon fractions.

As there is no additional info in the `.dat` file, would anyone be able to double check with their source? It would also be useful to include additional header comments in the `.dat` file. In the case those fractions are indeed baryons, it would be good to add a `gas` column, a 'stars' fraction column and leave the `baryon` columns as is. If someone has access to the error bars, it would be great to add them too, as the plot suggests those are quite significant.

If it would be useful, I can provide plots and scripts to reproduce the behavior. Thanks in advance!

**Comment by JBorrow** [comment]
If you could provide the figures that you referenced in the description that would be helpful, thanks.

**Comment by edoaltamura** [comment]
This is the plot of the data in Eckert2016 as process by the conversion script.
![image](https://user-images.githubusercontent.com/38359901/108192435-8afc3080-7114-11eb-99a3-f1820a1eca84.png)

Notice how in this figure there are many points above the 0.14 line, while in the paper (figure 4) all points are below that level. The factor from cosmology conversion is just a little > 1 and doesn't bring all points below the 0.14 line. If you look at figure 7 in the paper, however, the *baryon* fraction shifts the median relation up, which means that the points must on average shift up, presumably some will be above 0.14 as in my plot. I hope this helps.

**Comment by JBorrow** [comment]
I believe @MatthieuSchaller added this data; maybe he has comments here.

**Comment by MatthieuSchaller** [comment]
Yes, it says in the description of the data that it's a baryon content.

**Comment by edoaltamura** [comment]
Thanks - would it be possible to get the gas fractions too, for consistency with the rest of the datasets in the `HaloMassGasFractions` directory?

**Comment by MatthieuSchaller** [comment]
Do you have the data in a table somewhere?

**Comment by edoaltamura** [comment]
Unfortunately I don't. But if anyone could please contact the source of the current dataset and CC me I'd be happy to take over the conversation.

**Comment by MatthieuSchaller** [comment]
Actually @Moyoxkit added that data. Roi, could you remind us where you got it from? The "raw" data does not match the table in the paper.

**Comment by Moyoxkit** [comment]
I got this data from Stijn directly, so it is possible that it is wrong. I did not check it myself.

**Comment by MatthieuSchaller** [comment]
Ok. These are gas fractions under assumption of hydrostatic equilibrium. These have been re-analysed by Ian to improve the mass estimators from WL.

**Comment by edoaltamura** [comment]
That makes sense - thanks for checking. So they should be the data used to produce the fits in figure 10 of their paper, correct? For other people using these data, it would be useful to add this info in the convert script or in the dat file. @MatthieuSchaller do you  know if Ian also computed the error bars (in x and y)? It looks like they use them in all their fits.
